openmrs-compose.yaml

```diff
@@ -33,7 +33,7 @@ services:
     volumes:
       - ./utils/mysql.cnf:/etc/mysql/conf.d/custom.cnf  # mysql config preconfigured to allow binlog/debezium
       #- ./data:/var/lib/mysql
-      #- ./utils/dbdump:/docker-entrypoint-initdb.d # will execute any sql dumped here
+      - ./utils/dbdump:/docker-entrypoint-initdb.d # will execute any sql dumped here
```

Comment:
@kimaina can you please remind me what has changed here since last time that I tried `backup.sql` and it kept failing? The only change that I see (compared to that version that I tried) is the above comment line, i.e., not mapping the `/var/lib/mysql` directory but I don't think that will address the `liquibase` issues, right?

Reply:
Yes, this does not address the liquid base issues. See line 53/54